compose.yml

```diff
@@ -1,35 +1,36 @@
 services:
   mongodb:
     image: mongo:6.0.13
-    command: mongod --port ${MONGO_PORT:-27017}
     networks:
       - cdp-tenant
     ports:
-      - "${MONGO_PORT:-27017}:${MONGO_PORT:-27017}"
+      - "${MONGO_PORT:-27017}:27017"
```

Comment:
The internal container port MongoDB uses doesn't have to change as it will never be taken. We simply want to make sure the container's internal port is not mapped to a port that is taken on the host.